VMBackup/main/ScriptRunner.py

```diff
@@ -81,16 +81,25 @@ def get_config(self):
         try:
             with open(self.configLocation, 'r') as configFile:
                 configData = json.load(configFile)
```

Comment:
configData is under with block. is it not required to declare outside to access it outside ?